core/src/test/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotServiceSpec.scala

```
@@ -239,10 +242,198 @@ class SnapshotServiceSpec extends AnyWordSpecLike with Matchers with MockitoSuga
+              .additionalData(List(new WsmPolicyPair().key("group").value("additionalPolicyGroup")).asJava)
+          ).asJava
+        )
+
+        val mockWorkspaceManagerDAO = defaultMockWorkspaceManagerDao()
+        when(mockWorkspaceManagerDAO.getWorkspace(any[UUID], any[RawlsRequestContext])).thenReturn(
+          new WorkspaceDescription().stage(WorkspaceStageModel.RAWLS_WORKSPACE).policies(existingPolicyInputs.getInputs)
+        )
+
+        val mockDataRepoDAO: DataRepoDAO = new MockDataRepoDAO("mockDataRepo")
+
+        val workspace = minimalTestData.workspace
+
+        val snapshotService = SnapshotService.constructor(
+          slickDataSource,
+          mockSamDAO,
+          mockWorkspaceManagerDAO,
+          "fake-terra-data-repo-dev",
+          mockDataRepoDAO
+        )(testContext)
+
+        // call createSnapshot on the service
+        Await.result(
+          snapshotService.createSnapshot(workspace.toWorkspaceName,
+                                         NamedDataRepoSnapshot(DataReferenceName("foo"),
+                                                               Option(DataReferenceDescriptionField("foo")),
+                                                               UUID.randomUUID()
+                                         )
+          ),
+          Duration.Inf
+        )
+
+        // assert that the service checked to see if the workspace exists
+        verify(mockWorkspaceManagerDAO).getWorkspace(
+          ArgumentMatchers.eq(workspace.workspaceIdAsUUID),
+          any[RawlsRequestContext]
+        )
+
+        // assert that the service DID NOT call WSM's createWorkspace
+        verify(mockWorkspaceManagerDAO, times(0)).createWorkspace(
+          any[UUID],
+          ArgumentMatchers.eq(WorkspaceType.RawlsWorkspace),
+          any[Option[WsmPolicyInputs]],
+          any[RawlsRequestContext]
+        )
+
+        // assert that the workspace's existing policies were not updated
```

I'm trying to understand why we don't want them to be updated.. shouldn't the new policy just merge in?

If the answer is no because it would be an error case, then it seems like we should error if we encounter this, not just silently ignore the new policy.

---

We don't want it to be updated because it's not Rawls's place to modify existing policies on a workspace in WSM (rawls-stage or MC).